spec/uploaders/validate_file_size_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ValidateFileSize do
+  class ValidateFileSizeTest < CarrierWave::Uploader::Base
+    include ValidateFileSize
+
+    MAX_FILE_SIZE = 100
```

Comment:
To make this more uniform with Annas PR, can you make the MAX_FILE_SIZE a class method instead of a constant. See [here](https://github.com/department-of-veterans-affairs/vets-api/pull/5059/files#diff-5e5224c9764baaff6de004f124c5434cL10)

Reply:
I feel like that's out of scope for this change.  It might even make sense as it's own PR

Reply:
So.... I think the solution is to get rid of the ValidateFileSize module altogether.  Carrierwave has [a size range method](https://github.com/carrierwaveuploader/carrierwave/blob/f15f4ba71dd2c78e8df494bf65938bbaf13960f9/lib/carrierwave/uploader/file_size.rb#L21)

Reply:
Oh, good thought Anna. We can probably break this work out into a separate PR

Reply:
Good call! FWIW, I ran the specs for our two branches together and this was the only test to fail.  I can fix the conflict after you merge.

Reply:
I'll go forward with the merge here, then.